repose-aggregator/services/atom-feed/impl/src/main/resources/META-INF/schema/config/atom-feed-service.xsd

```diff
@@ -0,0 +1,189 @@
+           xmlns:html="http://www.w3.org/1999/xhtml"
+        >
+
+    <xs:element name="atom-feed-service" type="AtomFeedServiceConfigType"/>
+
+    <xs:complexType name="AtomFeedServiceConfigType">
+        <xs:sequence>
+            <xs:element name="feed" type="AtomFeedConfigType" minOccurs="1" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="AtomFeedConfigType">
+        <xs:all minOccurs="0">
+            <xs:element name="authentication" type="AuthenticationType">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            Defines the authentication mechanism to be used, and provides any necessary configuration.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:all>
+
+        <xs:attribute name="id" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines a unique ID which may be used to identify this feed.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="uri" type="xs:anyURI" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines the URI at which this feed can be read.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="polling-frequency" type="PositiveIntType" use="optional" default="5">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines how often, in seconds, the Atom Feed service checks this feed for updates.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="entry-order" type="EntryOrderType" use="optional" default="update">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Defines the order in which new Atom events will be delivered to any listeners registered
+                        on this feed.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:simpleType name="EntryOrderType">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="random">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            With this ordering, a feed listener will receive events with no ordering guarantees.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+
+            <xs:enumeration value="update">
```

Comment:
For whatever reason, that seems to be absent. My mistake.

So looking into this a bit today, here's some information I found in the Atom Spec (RFC 4287):
- This specification assigns no significance to the order of atom:entry elements within the feed.
- If multiple atom:entry elements with the same atom:id value appear in
   an Atom Feed Document, they represent the same entry.  Their
   atom:updated timestamps SHOULD be different.  If an Atom Feed
   Document contains multiple entries with the same atom:id, Atom
   Processors MAY choose to display all of them or some subset of them.
(From the above two points, it seems as though the order of Atom entries (defined by their id) may change over the span of multiple reads. If that is true, the id alone is not a sufficient field for the high water mark. Could/must links be used?)
- atom:entry elements MUST contain exactly one atom:updated element. 
- atom:entry elements MUST NOT contain more than one atom:published element.
(From the above two points, if we're looking at time, we can only guarantee ordering using the updated field)